docs/modules/ROOT/pages/backwards-compatibility.adoc

```diff
@@ -0,0 +1,35 @@
+= Backwards Compatibility
+
+OpenZeppelin Contracts uses semantic versioning to communicate backwards compatibility of its API and storage layout. Patch and minor updates will generally be backwards compatible, with rare exceptions as detailed below. Major updates should be assumed incompatible with previous releases. On this page, we provide details about these guarantees.
+
+Bear in mind that while releasing versions under `v1.0.0`, we treat minors as majors and patches as minors, in accordance with semantic versioning. This means that `v0.7.1` could be adding features to `v0.7.0`, while `v0.8.0` would be considered a breaking release.
+
+== API
+
+In backwards compatible releases, all changes should be either additions or modifications to internal implementation details. Most code should continue to compile and behave as expected. The exceptions to this rule are listed below.
+
+=== Security
+
+Infrequently a patch or minor update will remove or change an API in a breaking way, but only if the previous API is considered insecure. These breaking changes will be noted in the changelog and release notes, and published along with a security advisory.
+
+=== Underscores
+
+Functions and struct members with an underscore prefix should be considered "private" and may break in minor versions.
+
+=== Errors
+
+The specific error format and data that is included with reverts should not be assumed stable unless otherwise specified.
+
+=== Major Releases
```

Comment:
Are we changing to this format? We have been using normal sentences without capitalizing outside of the first word in headers. I see page titles changed, but also these inner sections. Shouldn't it be "Major releases"?

Reply:
This applies to the rest of the PR as well.

Reply:
fwiw, I liked the "Major releases" already, but is probably because I'm already used to it :).

Reply:
> Shouldn't it be "Major releases"?

+1

> fwiw, I liked the "Major releases" already, but is probably because I'm already used to it :).

Haha I actually prefer "Major Releases" (title case). We'd have to update the whole doc site though

Reply:
well this actually comes from the Solidity one, where I believe this is the style. Are we sure we're consistent with this in the rest of the docs for Cairo?

Reply:
It's consistent with the subheadings in the cairo docs. The two outliers for headings are "Interfaces and Dispatchers" and "Access Control"...it'd be consistent if we kept the title case for regular headings (as initially proposed in this PR)